recipe/meta.yaml

```
@@ -20,15 +20,23 @@ source:
     # qiskit-aer builds against a bundled blas on Windows. This patch adds a
     # USE_BUNDLED_BLAS_WIN cmake option to search for an installed blas library
     # instead
+    # This patch was accepted in https://github.com/Qiskit/qiskit-aer/pull/1940
+    # and can be removed from the receipe with the 0.14 release.
     - 0001-windows-find-blas.patch
+    # qiskit-aer uses an API from unistd.h to determine the total system memory
+    # at runtime. This works for Linux and for macOS when using the GitHub
+    # Actions C toolchain, but it does not work with the conda-forge C
+    # toolchain. This patch uses a different API. It was submitted upstream in
+    # https://github.com/Qiskit/qiskit-aer/pull/2016.
+    - 0001-Use-Apple-specific-API-to-determine-system-memory-on.patch
```

nitpick: maybe we don't need to patch for other OSs?
```suggestion
    - 0001-Use-Apple-specific-API-to-determine-system-memory-on.patch  # [osx]
```

---

(Either we keep the status quo, and bump the build number, or we limit the patch applicability and keep the same build number.)

---

Oh, yes, I forgot about the build number after resetting this branch back to `main`. We can limit the patch to osx. Since the CI is pretty far along, I will let it finish to see if there are any other issues before updating the branch.

---

Yup let's let it finish, no rush